### PR TITLE
Don't change source code

### PR DESCRIPTION
**Pull request by garyanikin**
Prevent changing source code (don't add unnecessary spaces inside comments)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1 +1 @@
-module.exports = require('./dist/less-syntax');
+module.exports = require('./lib/less-syntax');
```
Comment (mrmlnc) [inline index.js]: Why? See https://github.com/webschik/postcss-less/blob/master/package.json#L12

Comment (garyanikin) [inline index.js]: But there is no such directory

Comment (mrmlnc) [inline index.js]: These files will be generated Gulp. See https://github.com/webschik/postcss-less/blob/master/gulpfile.babel.js#L25

Comment (shellscape) [inline index.js]: I'm going to merge this PR and then revert this particular change

**Pull request merged**